mobly/base_suite.py

```diff
@@ -95,3 +95,38 @@ def setup_suite(self, config):
   def teardown_suite(self):
     """Function used to add post tests cleanup tasks (optional)."""
     pass
+
+  # Optional interfaces that users can override to record customized suite
+  # information to test summary.
+
+  def get_suite_name(self):
```

Comment:
shouldn't this just be part of the suite info?
why should it be a separate getter method?

Reply:
Test suites can return any custom values in `suite info`, our infra won't parse it and just write it to summary file.

So I created separate getter methods for fields that our infra needs to parse.